modules/ROOT/pages/x12-edi/x12-edi-connector.adoc

```diff
@@ -1,14 +1,15 @@
-= X12 EDI Connector
+= X12 EDI Connector - Mule 4
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
+:imagesdir: ../../assets/images/
 
-_Premium_
+Support Category: Premium
+X12 EDI Connector Version 2.1.1
 
-The X12 EDI Connector lets you convert X12 messages to and from
-DataWeave-compatible representations using lists and maps. X12 versions 003010, 003020, 003030, 003040, 003050, 003060, 003070, 004010, 004020, 004030, 004040, 004050, 004060, 005010, 005020, 005030, 005040, 005050, and 006020 are supported with supplied schema definitions for all standard transaction sets.
-Many HIPAA document versions are also supported, as listed separately in the
-Connector HIPAA reference page.
+The Anypoint X12 EDI Connector lets you convert X12 messages to and from DataWeave-compatible representations using lists and maps.
```

Comment:
Per Kim, naming convention for connectors is:
Anypoint Connector for X12 EDI
Then short version X12 EDI Connector

Reply:
Not in titles, got it.